README.md

```diff
@@ -102,8 +102,19 @@ server. (Yes, it's "OAUTHLIB" this time, not "AUTHLIB" as before.)
 $ export OAUTHLIB_INSECURE_TRANSPORT=1
 ```
 
-### Importing UCRs using Redis and Celery
 
+### Logging in as a local admin user
+
+There might be situations where you need to log into Superset as a local
+admin user, for example, to add a database connection. To enable local
+user authentication, in `superset_config.py`, set
+`AUTH_TYPE = AUTH_DB`.
+
+To return to allowing CommCare HQ users to log in, set it back to
+`AUTH_TYPE = AUTH_OAUTH`.
```

Comment:
nit: this can't be done in a production environment right? This is only for local setup?

Reply:
:thinking: Doing this in a production environment will prevent CommCare HQ users from logging in. But for self-hosted environments it could be useful for maintenance or setting up additional databases, etc.

Do you think I should add more context? Or maybe give advice on when to use CommCare Ansible, and the rare cases when it might be appropriate to edit the config directly?

Reply:
>  Doing this in a production environment will prevent CommCare HQ users from logging in

I believe that is true hence this should not be used for production environments.

I was very confused when I saw this for the first time. For me, I have always thought of this as something to be used only locally.
Though now when you mention maintenance, I see there is no way to access UI for an admin without this change, so it is the only way out.

> the rare cases when it might be appropriate to edit the config directly

Right, we should just caution that doing this change disables usual new user login (I assume the ones who are logged in will continue to work), so, on a production environment this should be only done when system isn't in use.

P.S: why do we need this change though? I comment about that would be super helpful in the code

Reply:
ceb8f80